raiden/tests/integration/api/test_restapi.py

```diff
@@ -8,16 +8,17 @@
     to_canonical_address,
     is_checksum_address,
 )
-from raiden_contracts.constants import CONTRACT_HUMAN_STANDARD_TOKEN, MAX_TOKENS_DEPLOY
+from raiden_contracts.constants import (
+    CONTRACT_HUMAN_STANDARD_TOKEN,
+    MAX_TOKENS_DEPLOY,
+    TEST_SETTLE_TIMEOUT_MIN,
+    TEST_SETTLE_TIMEOUT_MAX,
```

Comment:
what is guaranteeing that the raiden repo is deploying the smart contracts with the same constants for min and max settlement?

Reply:
As we deploy with these values I thinks it's fine.

Reply:
We don't have any place where we actually write down the values? A production deployment script? Because the values decided in https://github.com/raiden-network/spec/issues/16 are buried in a closed issue.